src/app/tabs/debugger/debuggerUI/vmDebugger/utils/SolidityTypeFormatter.js

```diff
@@ -1,18 +1,21 @@
 var yo = require('yo-yo')
 var BN = require('ethereumjs-util').BN
 
+/* jslint eqeqeq: false */
+/* jslint eqeq: false */
+
 module.exports = {
   formatSelf: formatSelf,
   extractData: extractData
 }
 
 function formatSelf (key, data) {
   var style = fontColor(data)
-  var keyStyle = data.isProperty ? 'color:#847979' : ''
+  var keyStyle = data.isProperty ? 'color: var(--info)' : ''
   if (data.type === 'string') {
     data.self = JSON.stringify(data.self)
   }
-  return yo`<label style=${keyStyle}>${key}: <label style=${style}>${data.self}</label><label style='font-style:italic'> ${data.isProperty || !data.type ? '' : ' ' + data.type}</label></label>`
+  return yo `<label style='${keyStyle};white-space:pre-wrap;'> ${' ' + key}:<label style=${style}>${' ' + data.self}</label><label style='font-style:italic'> ${data.isProperty || !data.type ? '' : ' ' + data.type}</label></label>`
```

Comment:
I cannot read this line. Use classes instead of setting the style directly, also format the code.
Also I don't understand why doing : `${' ' + key}`, this shoud be ` ${key}`

Reply:
this should not be here.